### PR TITLE
faster add child #3152

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
No new arrays being created here! Also tweaked the eslintsrc to   allow
us to not have to use rest.

### CODE REVIEW

File: src/core/display/Container.js [hunk 1]
```diff
@@ -64,18 +74,14 @@ export default class Container extends DisplayObject
 
             child.parent = this;
 
-            // ensure a transform will be recalculated..
-            this.transform._parentID = -1;
```
Comment (englercj) [inline src/core/display/Container.js]: Why was this removed? Pretty sure it is necessary to reset the transform/bounds Ids when adding the child to a new parent.

**Comment by GoodBoyDigital** [comment]
Arrrg! Where would we be without your eagle eyes! Great catch dude

**Comment by englercj** [comment]
Fixed!
